core/src/main/java/org/apache/struts2/components/Set.java

```diff
@@ -136,6 +138,16 @@ public void setValue(String value) {
         this.value = value;
     }
 
+    @StrutsTagAttribute(description="Set to true to prevent the default whitespace-trim of this tag's body content")
+    public void setDoNotTrimBody(String doNotTrimBody) {
```

Comment:
You should use `boolean` with correct `type` set in annotation. Take a look how it is done in `Property` and `PropertyTag` for example.

Reply:
This has been addressed in the most recent commit (the setters and tag annotation now use boolean, as requested).